CONTRIBUTING.md

```diff
@@ -113,84 +117,71 @@ It is a good idea to create a detailed (but concise) comment to accompany the in
+Periodically, the current lead developer(s) of Nucleus Coop will decide when enough changes have been incorporated into the master branch to justify a new release. To ensure build stability, a release will be created and published alongside a change log of implemented changes after sufficient testing by maintainers. Releases include source code along with a more convenient installer and a "portable" (pre-compiled zip folder) option for users to run the program.
 
 Currently, there is no set release schedule, and releases will be published when ready and verified stable.
 
 ### Code Standards
-* When in doubt, structure your code to err on the side of readability!
+* When in doubt, structure your code to prioritize readability!
 * Comment large blocks of abstract code with concise descriptions.
 * Please write C# code following [standard conventions](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions). 
 
 ### Tips and Troubleshooting
 
 #### Updating Submodules
-If you get an error about missing URL's when attempting to initalize and update the submodules, open up .gitmodules (in the project root folder) in a text editor and make sure that the following are listed.
+If you get an error about missing URL's when attempting to initalize and update the submodules, open up .gitmodules (in the project root folder) in a text editor and make sure that the following are listed. 
     
-Even if some of these dependencies are not in active use, the git "update" command may fail to execute if entries and URLs for **all submodule** dependencies cloned from your fork are not found in the .gitmodules file.
-
-    [submodule "Submodules/x360ce"]
-	    path = Submodules/x360ce
-	    url = https://github.com/lucasassislar/x360ce
-
-    [submodule "Submodules/ProtoInput"]
-        path = Submodules/ProtoInput
-        url = https://github.com/ilyaki/protoinput
-
-    [submodule "Submodules/nukeupdater"]
-        path = Submodules/nukeupdater
-        url = https://github.com/lucasassislar/nukeupdater
-        
-    [submodule "Submodules/TypescriptSyntaxPaste"]
-        path = Submodules/TypescriptSyntaxPaste
-        url = https://github.com/nhabuiduc/TypescriptSyntaxPaste.git
+Even if some of these dependencies are not in active use, the git "update" command may fail to execute if entries and URLs for **all submodule** dependencies cloned from your fork are not found in the .gitmodules file. Add anything missing, save the changes to .gitmodules and the command should now work.
+```

```

Comment:
to get code highlighting :)
```suggestion
```ini
```

It should then look like this:
```ini
[submodule "Submodules/x360ce"]
    path = Submodules/x360ce
    url = https://github.com/lucasassislar/x360ce
[submodule "Submodules/ProtoInput"]
    path = Submodules/ProtoInput
    url = https://github.com/ilyaki/protoinput
[submodule "Submodules/nukeupdater"]
    path = Submodules/nukeupdater
    url = https://github.com/lucasassislar/nukeupdater
    
[submodule "Submodules/TypescriptSyntaxPaste"]
    path = Submodules/TypescriptSyntaxPaste
    url = https://github.com/nhabuiduc/TypescriptSyntaxPaste.git
```